frontend/src/v5/store/users/users.redux.ts

```diff
@@ -19,11 +19,14 @@ import { createActions, createReducer } from 'reduxsauce';
 import { Constants } from '../../helpers/actions.helper';
 
 export interface IUser {
-	user: string;
+	username: string;
 	firstName: string;
 	lastName: string;
 	company?: string;
 	job?: string;
+	email?: string;
+	hasAvatar?: boolean;
+	avatarUrl: string;
```

Comment:
This is from the v4 schema . Maybe we need to rename IUser to IMember to reflect the difference, and add a complete IUser for the full user.

Reply:
Or change the backend to return a more complete user and have the same schema.

Reply:
Renaming `IUser` to `IMember` would be nice, especially because accessing the property `user.user` does not feel right